test/functional/tools/secret_tool.xml

```diff
@@ -0,0 +1,15 @@
+<tool id="secret_tool" name="secret_tool" version="test" profile="23.0">
+    <requirements>
+        <credentials name="service1" reference="service/54687" optional="false" multiple="true" label="Your credentials set" description="Optional description of the service using credentials">
```

Comment:
Wondering about `multiple="true"` here: how would the environment variable names be constructed if multiple credentials are given?